pkg/scheduler/api/types.go

```diff
@@ -19,7 +19,7 @@ package api
 import (
 	"time"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
nit: corev1 would be more expressive and consistent with the rest.

Reply:
actually I ended up not changing in this file, will revert